packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -26,6 +26,8 @@ import {
   ZoeMintI,
   ZoeStorageManagerIKit,
 } from '../typeGuards.js';
+import '../internal-types.js';
+import './internal-types.js';
```

Comment:
I'm getting skeptical these are all necessary :)

Reply:
Of all of these, only the second import above is necessary. I do not understand why, but without it client modules like governance then claim that the `../zoe` package has many "Cannot find name <type>" errors for other types in the zoe package.

So the other one is gone. Resolving.